code/modules/projectiles/projectile/bullets.dm

```diff
@@ -415,6 +415,13 @@
 	name = "practice"
 	damage = 5
 
+//4mm. Tiny, very low damage, does not embed, but has very high penetration. Only to be used for the experimental SMG.
+/obj/item/projectile/bullet/proto_smg
+	damage = 23
+	penetrating = 1
+	armor_penetration = 40
```

Comment:
in what world is penetrating 1, 23 damage and 40 AP "very low damage"?

Reply:
Resolved on Discord, going with different values